### DISCUSSION TITLE
🐱‍🏍 技术预览版发布

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#58] [展示]
## 新设计

1. 使用 Mica 材质作为基底，搭配半透明的卡片材质，极大提升沉浸感。
2. 全面应用圆角设计，界面观看更为友好。
3. 自适应设计，不同宽度下都能有不错的观感。
4. 悬浮卡片替代 v1 的副页，提供更加优秀的操作体验

## 新功能

1. 支持查看 `热门` 视频
2. 支持观看 `直播`
3. 番剧/国创大升级
4. 独立的专栏探索
5. 纪录片/电影/电视剧等内容更丰富，且支持索引

哔哩哔哩( ゜- ゜)つロ干杯~

![截图](https://img13.360buyimg.com/ddimg/jfs/t1/120826/9/15566/1348721/614490ffE0a727387/037f639ee686276e.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2109.1701.pre-release'>技术预览版</a>.</em>

**Comment by chenaichenet** [comment]
OHHHHHHH，哔哩哔哩用户前来支持

**Comment by NriotHrreion** [comment]
前来支持！
